todo_app/app.py

```diff
@@ -1,15 +1,47 @@
-from flask import Flask
-
+from flask import Flask, render_template, request, redirect, url_for
 from todo_app.flask_config import Config
 
+from todo_app.data.session_items import get_items, add_item, save_item, get_item, remove_item
+
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', todoList = get_items())
+
+@app.route('/new_todo', methods=['POST'])
+def add_item_from_form():
+    title = request.form['title']
+    add_item(title)
+    return redirect(url_for('index'))
+
+@app.route('/update_todo/<id>', methods=['POST'])
+def update_item(id):
+    ticked = False
+    try:
+        if request.form['completed'] == 'on':
+            ticked = True 
+    finally:
+        item = get_item(id)
+        if ticked:
+            item['status'] = 'Completed'
+        else:
+            item['status'] = 'Not Started'
+        save_item(item)
+        return redirect(url_for('index'))
+
+@app.route('/sort_status', methods=['GET'])
```

Comment:
It would be a bit neater to have a query parameter for the '/' endpoint rather than having two routes that load the same page with a small difference like sorting.

Reply:
Refactored as per suggestion - I added another <A> tag for descending order.